spec/jobs/upvs/fetch_public_authority_edesks_list_job_spec.rb

```diff
@@ -23,7 +23,7 @@
 
     it 'supports semicolon separated format' do
       downloader = double
-      expect(downloader).to receive(:download_file).with('https://url').and_return(fixture_filepath('jobs/upvs/fixtures/upvs-edesks-v2.csv'))
+      expect(downloader).to receive(:download_file).with('https://url').and_return(fixture_filepath('fixtures/upvs/upvs-edesks-v2.csv'))
```

Comment:
`fixture_filepath('fixtures/upvs/upvs-edesks-v2.csv')`

by som cakal ze bude

`fixture_filepath('upvs/upvs-edesks-v2.csv')`

lebo inak je to `spec_filepath`

Reply:
mas pravdu, upravim